### ISSUE TITLE
Prepare Project 

### ISSUE THREAD

**Issue by PexegoUva** [PexegoUva/project-lambda#1]
### :tophat: What is the goal?

To provide tasks that will be attached to Issues that will define how I'm going to work developing the project.

### :memo: How is it being implemented?

First of all, I need to prepare templates for **Issues** and **Pull Requests** (bugs, features, refactor, pull requests). Once all of them ready just generate a new PR to merge templates so we will be able to create and assign issues to the tasks.

### :clipboard: Tasks to implement the Issue
 - [x] Document Architectural decisions (explain that in the wiki + add link from Readme) - #4 
 - [x] Research and document how to fake API (third party tool, own code etc) - #5
 - [x] Initialize Android project and upload it to Github - #6
 - [x] Prepare CI/CD before starting to code (travis file, badges, extra needed tools for the process) - #7
 - [x] Document tools (detekt, codecov, jacoco) in the wiki + add links in the Readme - #7
 - [x] Init visuals + develop SplashScreen and LoginScreen (add navigator to travel between views) - #8
 - [x] Add MVP for the login - #9 
 - [x] Connect Presenter to the PostLogin UseCase - #10
 - [x] Connect PostLogin UseCase to the Repository (fake API calls) - #11
 - [x] Generate first release, explain where to find APK in the Readme - #12
 - [x] Add Logout screen (navigator taking care of existing session) - #13 
 - [x] Add MVP and connect Presenter to the PostLogout UseCase - #14
 - [x] Add Repository part with the DB (remove existing token) - #15
 - [x] Release the final version, check missing content - #16

### :notebook: Notes Related to the task

- Each task will be attached to an Issue that will be linked to the current Project

**Comment by PexegoUva** [comment]
Default **Issue** and **Pull Request** templates are ready to roll!!

**Comment by PexegoUva** [comment]
Assign first task to #4 issue

**Comment by PexegoUva** [comment]
Assign second task to #5, third task to #6 and tasks four and five to #7

**Comment by PexegoUva** [comment]
Finished to assign Sprint 1 Issues, let's start with the second sprint tasks

**Comment by PexegoUva** [comment]
Add tasks 6, 7, 8 9 and 10 to their respective issues. Finished Sprint 2 definitions

**Comment by PexegoUva** [comment]
Finished the third Sprint Issues by adding 11 to 14 tasks and their issues. Now I'm ready to begging the development of the APP

**Comment by PexegoUva** [comment]
Added missed mockups to the #8 and #13 issues
